### ISSUE TITLE
Cannot read property 'code' of undefined

### ISSUE THREAD

**Issue by amilbeck** [MrCodingB/Postman-Collection-Explorer-VS-Code#12]
**Describe the bug**
I just installed this extension today. I was able to finally get it to recognize my collection after tinkering with the regex a bit. Opened a test script for one of my requests and edited a few lines. I then tried to send a request and I keep getting the following error:

```
Error running command postman-collection-explorer.sendRequest: Cannot read property 'code' of undefined. This is likely caused by the extension that contributes postman-collection-explorer.sendRequest.
```

I have no idea what this means or how to fix it. I've tried closing VS Code and opening again. Even tried reloading the collection in the workspace. I also unchecked strict SSL because I wasn't sure if that was the problem or not but that didn't change anything either. Any idea what I'm doing wrong here? 

**Screenshots**
![image](https://user-images.githubusercontent.com/38883415/151416655-72c7a13d-151a-43ca-a649-8d045c2951a3.png)

**Environment (please complete the following information):**
 - OS: Windows
 - VS Code Version: 1.63.2 
 - Version: 0.2.2

**Comment by MrCodingB** [comment]
What sort of request did you run? I know how to fix the issue, but I'd like to understand what caused it, because the access `code` should never throw an error at that point, the object isn't supposed to ever be undefined.

**Comment by MrCodingB** [comment]
Can you try, if the changes in my branch fix your issue?

**Comment by amilbeck** [comment]
> Can you try, if the changes in my branch fix your issue?

Just now noticing this. Sorry it took so long. I guess I had notifications turned off. I will try it out later today and let you know.

**Comment by MrCodingB** [comment]
I think though, that the real problem isn't the extension, but your API. Because this error could have only been thrown, if the request that you triggered didn't get a response from the server.

**Comment by MrCodingB** [comment]
This is fixed in v0.3.1
